src/nativestore/CentralPropertyLink.cpp

```diff
@@ -0,0 +1,224 @@
+/**
+Copyright 2023 JasminGraph Team
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+ */
+
+#include "CentralPropertyLink.h"
+
+#include <sstream>
+#include <vector>
+#include <iostream>
+
+#include "../util/logger/Logger.h"
+
+Logger central_property_link_logger;
+unsigned int CentralPropertyLink::nextPropertyIndex = 1;  // Starting with 1 because of the 0 and '\0' differentiation issue
+std::string CentralPropertyLink::DB_PATH = "/home/ubuntu/software/jasminegraph/streamStore/properties.db";
```

Comment:
This value should not be a hard coded value, but rather get it from the properties file as a configurable property.